Rakefile

```diff
@@ -59,3 +59,10 @@ namespace :tests do
     spec.call('./spec/api/*_spec.rb')
   end
 end
+namespace :assets do
+  desc "Update the routes metadata"
+  task :precompile do
+    sh 'grep -rh "# route" api > routes.tmp'
+    sh 'roda-parse_routes -f routes.json routes.tmp'
```

Comment:
If we would add a `jq . < routes.json` we get something thats easier to read for humans :)

Reply:
$ jq
sh.exe": jq: command not found

I don't like humans, except jeremyevans, of course, XD.

How does jq . looks like? I can make a ruby oneliner with the same output

Reply:
> I don't like humans, except jeremyevans, of course, XD.

:laughing: 

jq is only available when it is installed of course. It personally my favorite tool to quickly view json files in the terminal.

But I don't insist on it :)